### PR TITLE
Browse addons and random uploads

### PR DESCRIPTION
**Pull request by mstriemer**
This will browse around the site 5 times more often than uploading an add-on. The URLs it visits are fairly consistent it might be better to pick links to click at random but I wasn't sure what we'd end up with there.

This will also start picking one of the XPIs in the add-ons folder at random when uploading.

### CODE REVIEW

**Comment by mstriemer** [comment]
r? @kumar303 

Any concerns on the browsing bit?

File: locustfile.py [hunk 1]
```diff
@@ -164,6 +171,17 @@ def upload(self):
         if form:
             self.upload_addon(form)
 
+    @task(5)
```
Comment (kumar303) [inline locustfile.py:174]: I think 5 times more might be a bit much since we mainly want to stress uploads (for now). Maybe twice as much?

Comment (kumar303) [inline locustfile.py:174]: well, huh, I guess 5 times more would be more realistic so sure why not

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
+cssselect >= 0.9.1
```
Comment (kumar303) [inline requirements.txt:1]: hottt

File: locustfile.py [hunk 3]
```diff
@@ -164,6 +171,17 @@ def upload(self):
...
+    def browse(self):
+        self.client.get('/en-US/firefox/')
+        self.client.get('/en-US/firefox/search/?q=pi&appver=45.0&platform=mac')
+        with self.client.get(
+                '/en-US/firefox/extensions/',
+                allow_redirects=False, catch_response=True) as response:
```
Comment (kumar303) [inline locustfile.py:180]: when you use `catch_response` I think you have to call `response.failure()` manually otherwise it won't ever happen.

Comment (mstriemer) [inline locustfile.py:180]: Yeah this would totally throw an exception on an error. I'll update it.

**Pull request merged**